code/modules/mob/living/carbon/human/species_types/podpeople.dm

```diff
@@ -49,10 +49,10 @@
 		var/turf/T = H.loc
 		light_amount = min(1, T.get_lumcount()) - 0.5
 		H.adjust_nutrition(5 * light_amount * delta_time)
-		if(light_amount > 0.2) //if there's enough light, heal
+		/*if(light_amount > 0.2) //if there's enough light, heal
```

Comment:
I would rather we make the pod full powered subtype now instead of later. Just requires a boolean var healed_by_light, and the subtype should be used for the grown and ghost role.

Reply:
Could I simply override this particular proc under the subspecies I create? That way the roundstart podpeople don't even bother checking a var for healing.

Reply:
No, you would have to copypaste the whole thing if you did that.

Reply:
I've solved this by creating a new subtype, and splitting light healing off into its own proc that only does healing for the subtype.